packages/curvenote-cli/src/submissions/submit.utils.ts

```diff
@@ -85,7 +84,10 @@ export async function determineCollectionAndKind(
   collections: CollectionListingDTO,
   opts?: { kind?: string; collection?: string; yes?: boolean },
 ) {
-  const openCollections = collections.items.filter((c) => c.open);
+  const openCollections = [
+    ...collections.items.filter((c) => c.open && c.default),
```

Comment:
Default at the top - we could also make it bold or `... (default)` or something in the selection choices...